client/src/components/Tags.tsx

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
```suggestion
import React from 'react';
```

You should be importing react where ever JSX is used, where you have class components, and where you have functional components.